Engine/source/console/consoleObject.h

```diff
@@ -542,6 +545,19 @@ class AbstractClassRep : public ConsoleBaseType
 
 extern AbstractClassRep::FieldList sg_tempFieldList;
 
+inline AbstractClassRep* AbstractClassRep::getClassRepFromName(const char* className)
+{
+	AbstractClassRep* current = classLinkList;
+	while(current != NULL)
+	{
+		if(dStricmp(current->getClassName(),className) == 0)
```

Comment:
Since getClassName() returns a StringTableEntry it would be better to convert className to a StringTableEntry then you can just compare pointers instead of using a string compare.